package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "plasma-contracts",
+  "version": "1.0.0",
+  "description": "Root chain contracts for Plasma MVP, work in progress.",
```

Comment:
How about: MVP --> M(ore)VP

Reply:
I didn't explicitly add this - npm must pick it up automatically from the README!
I'll change it in both places 😄